madgraph5amcatnlo.spec

```diff
@@ -35,6 +35,18 @@ rm bin/compile.py
 # Save patched config
 mv input/mg5_configuration_patched.txt input/mg5_configuration.txt
 
+# Start small NLO event generation to make sure that all additional packages are compiled
+cat <<EOF > basiceventgeneration.txt
```

Comment:
Hello @davidlange6 ,
from https://herwig.hepforge.org/tutorials/installation/prerequisite.html

"Madgraph writes some files to the source location the first time it is run, which can cause problems if you install it on a read-only filesystem. We would therefore recommend you run a simple process during install to avoid this problem."

If we can get a proper library for this, it's also fine. We just need to have Mg5amc@NLO  runnin properly when we call it with Herwig7 matchbox

Just to give an update: I'm revising some changes to this PR right now to make Herwig 7.1.1 (newest version) and its interface run in CMSSW. This should be finished in the next few days.